substrate/frame/staking-async/src/pallet/impls.rs

```diff
@@ -856,6 +861,129 @@ impl<T: Config> Pallet<T> {
 	pub fn api_pending_rewards(era: EraIndex, account: T::AccountId) -> bool {
 		Eras::<T>::pending_rewards(era, &account)
 	}
+
+	/// Calculate the total stake of the lowest portion validators and store it for the planned era.
+	///
+	/// Removes the stale entry from [`EraLowestRatioTotalStake`] if it exists.
+	pub(crate) fn calculate_lowest_total_stake() {
+		// Only calculate if unbonding queue params have been set.
+		if let Some(params) = UnbondingQueueParams::<T>::get() {
+			// Determine the total stake from the lowest portion of validators and persist for the
+			// era.
+			let mut exposures: Vec<_> =
```

Comment:
This is not exposure, this is `exposure_total`, or better said, `total_stake`. I think using `exposure` is misleading as it might imply the type is `Exposure`.

Reply:
Changed in [aa3d40c](https://github.com/paritytech/polkadot-sdk/pull/8298/commits/aa3d40c2cb608696e9669c2a5f1bea022fd4848d).